lang/types.md

```diff
@@ -22,9 +22,9 @@ In the future, we might want to separate a type from its layout, and consider th
 ```rust
 /// A "layout" describes the shape of data in memory.
 struct Layout {
-    size: Size,
-    align: Align,
-    inhabited: bool,
+    pub size: Size,
```

Comment:
These fields are used in `BasicMemory::retag_ptr`.
We could also keep it private and add `.size()` etc. methods for Layout, but I think thats a bit verbose for a spec.